src/rng/Counter_RNG.hh

```diff
@@ -296,13 +288,8 @@ public:
   size_t size_bytes() const { return sizeof(data); }
 
 private:
-  mutable ctr_type::value_type data[CBRNG_DATA_SIZE];
-
-  //! Private copy constructor.
-  Counter_RNG(const Counter_RNG &);
-
-  //! Private assignment operator.
-  Counter_RNG &operator=(const Counter_RNG &);
+  //! \bug Can this be changed to std::array<ctr_type::value_type,CBRNG_DATA_SIZE> ?
+  mutable ctr_type::value_type data[CBRNG_DATA_SIZE]; // NOLINT
```

Comment:
Just a comment here--I think this was for pre-C++11 support in Random123?

Reply:
> Just a comment here--I think this was for pre-C++11 support in Random123?

I think you are correct.  But I didn't want to figure out how to convert the type to `std::array` right now.  We can revisit this in the future.